### ISSUE TITLE
GoogleDrive `redirect_uri=http://localhost:port/` should be `redirect_uri=http://localhost:port`

### ISSUE THREAD

**Issue by NikitaIT** [remotestorage/remotestorage.js#1212]
0. Open console `https://console.developers.google.com/apis/credentials/oauthclient/${your_client_id}?project=${your_project_number}`
1. Set `http://localhost:port` as allowed to redirect and scripts. (URL ends with "/" - not valid)
2. Go to `http://localhost:port`.
3. Select GoogleDrive in Widget.
4. App sends request like: `https://accounts.google.com/o/oauth2/auth?redirect_uri=http://localhost:port/&scope=https://www.googleapis.com/auth/drive&client_id=...&response_type=token`
5. Oops.
```
Error:
The redirect URI in the request, http://localhost:port/, does not match the ones authorized for the OAuth client.
```

P.s. If i send request manualy without "/" it works fine.

env:
    "remotestorage-widget": "^1.5.2",
    "remotestoragejs": "^2.0.0-alpha.5",

**Comment by raucao** [comment]
Just published `2.0.0-alpha.5` including the fix for this (and with a *much* smaller library file size).

**Comment by NikitaIT** [comment]
@raucao @galfert oh, i try it 2 days ago again, and i can add "/" in Authorized redirect URIs (although earlier I could not), and it works. May be its my fault. Or it fixed by google, i dont know. I suspect this might be a breaking change for someone. The situation seems very strange.
![image](https://user-images.githubusercontent.com/16810067/103167786-3b4d6700-483f-11eb-9e1e-645a00c3b53e.png)
